js/index.js

```diff
@@ -2,16 +2,42 @@ var React = require("react");
 var ReactDOM = require("react-dom");
```

Comment:
Purpose is to separate major functionalities into files, to ensure one file contains one meaningful part/logic structure.